src/coverage/inject/cli.cr

```diff
@@ -1,31 +1,39 @@
 require "option_parser"
+
 # require "tempfile"
 
 module Coverage
   module CLI
     def self.run
       output_format = "HtmlReport"
-      filenames = [] of String
+      targets = [] of String
       print_only = false
 
       OptionParser.parse! do |parser|
-        parser.banner = "Usage: crystal-cover [options] <filename>"
+        parser.banner = "Usage: crystal-coverage [options] <directories or filenames>"
+        parser.on("-h", "--help", "show this help") { puts parser; exit }
         parser.on("-o FORMAT", "--output-format=FORMAT", "The output format used (default: HtmlReport): HtmlReport, Coveralls ") { |f| output_format = f }
         parser.on("-p", "--print-only", "output the generated source code") { |_p| print_only = true }
         parser.on("--use-require=REQUIRE", "change the require of cover library in runtime") { |r| Coverage::SourceFile.use_require = r }
         parser.unknown_args do |args|
           args.each do
-            filenames << ARGV.shift
+            targets << ARGV.shift
           end
         end
       end
 
-      raise "You must choose a file to compile" unless filenames.any?
+      raise "You must choose at least one file to compile" unless targets.any?
```

Comment:
@anykeyh Maybe just printing this phrase is enough. Do you think it really have to be an Exception?

Reply:
We might use the new [Log](https://crystal-lang.org/api/0.34.0/Log.html) module to raise an Error message and exit the application instead?

Reply:
That's a good idea! I'll update it

Reply:
@anykeyh what do you think?